docs/source/notebooks/clv/sBG.ipynb

```diff
@@ -0,0 +1,2840 @@
+{
```

Comment:
<p>Maybe add comments on how to read or interpret these parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Done